low-level/framework/base/sspl_constants.py

```diff
@@ -107,6 +107,8 @@
 SB_DATA_PATH = DATA_PATH + 'support_bundle'
 SSPL_SB_TMP = f"{SB_DATA_PATH}/sspl_support_bundle_tmp"
 SSPL_SB_REQUEST = f"{SB_DATA_PATH}/sspl-sb-req.txt"
+SB_REQUEST_TIMEOUT = 1500
+SB_REQUEST_TRACKER_DIR = "/var/cortx/resources/"
```

Comment:
this would be defined in confstore already, which SB infra would have introduced ? We should be using same. Or any change there at common infra reference, wont reflect for SSPL SB

Reply:
Yes. now taking this from cortx.conf (confstore)
Done